src/OpenFeature.Contrib.Providers.Flagsmith/FlagsmithProvider.cs

```diff
@@ -65,17 +65,12 @@ public FlagsmithProvider(IFlagsmithProviderConfiguration providerOptions, IFlags
 
         private Task<IFlags> GetFlags(EvaluationContext ctx)
         {
-            string key = null;
-            if (ctx != null && ctx.TryGetValue(Configuration.TargetingKey, out var value))
-            {
-                key = value?.AsString;
-            }
+            var key = ctx?.TargetingKey;
```

Comment:
Related: https://github.com/open-feature/dotnet-sdk/issues/235

Though I think everything should be backwards compatible when we make this change.